libraries/Arduino_LED_Matrix/examples/DisplaySingleFrame/DisplaySingleFrame.ino

```diff
@@ -1,21 +1,39 @@
-#include "Arduino_LED_Matrix.h"
-#include "frames.h"
+/*
+Single Frame
```

Comment:
```suggestion
  Single Frame
```

Use consistent block comment style. The convention of indenting the contents of the multi-line comment by two spaces was established by the default configuration of the Arduino IDE 1.x Auto Format feature, and used in official Arduino code.